src/general/MainMenu.tscn

```diff
@@ -1052,3 +1059,5 @@ DialogText = "STEAM_INIT_FAILED_DESCRIPTION"
 [connection signal="OnFinishedSignal" from="CreditsView/CreditsScroll" to="." method="OnReturnFromCredits"]
 [connection signal="pressed" from="CreditsView/Back" to="." method="OnReturnFromCredits"]
 [connection signal="OnClosed" from="ModManager" to="." method="OnReturnFromMods"]
+[connection signal="Cancelled" from="LowPerformanceDialog" to="LowPerformanceDialog" method="OnConfirmed"]
```

Comment:
This seems like a hack to me. Could you instead edit the dismissible warning class to add a new property to make it remember the dismissed state on cancel? It should probably be an enum to allow flexibility later, the default should be `RememberOnConfirm` and here the value should be `RememberOnCancel`.